spec/classes/init_spec.rb

```diff
@@ -268,6 +269,27 @@
     end
   end
 
+  describe 'with \'fqdn_ip_address\' parameter set to' do
+    context 'an invalid type (not ip address)' do
```

Comment:
I tried to create it but didn't worked. I also found your comment regarding exported resources and thought it's not possible to define a positive test. I'll try again.
 # GH: rspec-puppet does not yet support checking for exported resources